<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1548</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">background-image: url(:/images/resources/ss.jpg);</string>
         </property>
         <widget class="QLabel" name="qr_label">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>340</y>
            <width>181</width>
            <height>171</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background:none;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border:none;
background:transparent;
}

QLabel {
	background:transparent;
    font-family: 'Arial';
    color: #333; /* Dark Gray */
}


QLineEdit{
background:transparent;
    border-radius: 5px;
    padding: 5px;
}
QLineEdit:focus {
    border-color: #008080; /* Teal */
}
Line{
background:transparent;
}
</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>266</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Email Address</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="password">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>360</y>
            <width>321</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="email">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>286</y>
            <width>321</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>190</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="name">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>210</y>
            <width>321</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>340</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>20</y>
            <width>171</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">text-align: center;</string>
          </property>
          <property name="text">
           <string>FortiFile</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>100</y>
            <width>221</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#767676;&quot;&gt;Create Account&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>234</y>
            <width>321</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="Line" name="line_2">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>310</y>
            <width>321</width>
            <height>16</height>
           </rect>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="Line" name="line_3">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>384</y>
            <width>321</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="next_btn">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>450</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #008080;  /* Teal */
    color: white;
    font-size: 18px;
    border-radius: 10px;
    padding: 8px 20px;
}
QPushButton:hover {
    background-color: #006666; /* Darker teal */
}
</string>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>25</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource.qrc">:/icons/resources/logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="error_icon_2">
          <property name="geometry">
           <rect>
            <x>186</x>
            <y>507</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource.qrc">:/icons/resources/icons8-error-50.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="error_text_2">
          <property name="geometry">
           <rect>
            <x>214</x>
            <y>510</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #D9534F;

 </string>
          </property>
          <property name="text">
           <string>All fields are required</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="lofin_btn">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>548</y>
            <width>93</width>
            <height>41</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border:none;
color: rgb(0, 134, 110);
background:transparent;
}


QPushButton:hover:!checked{
	color: rgb(0, 56, 46);
}</string>
          </property>
          <property name="text">
           <string>Log In</string>
          </property>
         </widget>
         <widget class="Line" name="line_5">
          <property name="geometry">
           <rect>
            <x>338</x>
            <y>568</y>
            <width>38</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>174</x>
            <y>560</y>
            <width>161</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Already have an account?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <zorder>line_5</zorder>
         <zorder>lofin_btn</zorder>
         <zorder>label_2</zorder>
         <zorder>password</zorder>
         <zorder>email</zorder>
         <zorder>label</zorder>
         <zorder>name</zorder>
         <zorder>label_3</zorder>
         <zorder>label_5</zorder>
         <zorder>label_7</zorder>
         <zorder>line</zorder>
         <zorder>line_2</zorder>
         <zorder>line_3</zorder>
         <zorder>next_btn</zorder>
         <zorder>label_10</zorder>
         <zorder>error_icon_2</zorder>
         <zorder>error_text_2</zorder>
         <zorder>label_12</zorder>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <widget class="QLineEdit" name="totp">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>310</y>
            <width>231</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
border:none;
background:transparent;
}</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>290</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Access Key</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>50</y>
            <width>171</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">text-align: center;</string>
          </property>
          <property name="text">
           <string>FortiFile</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>150</y>
            <width>221</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#767676;&quot;&gt;Verify&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="verify_btn">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>440</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #008080;  /* Teal */
    color: white;
    font-size: 18px;
    border-radius: 10px;
    padding: 8px 20px;
}
QPushButton:hover {
    background-color: #006666; /* Darker teal */
}
</string>
          </property>
          <property name="text">
           <string>Verify</string>
          </property>
         </widget>
         <widget class="Line" name="line_4">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>342</y>
            <width>231</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>540</y>
            <width>371</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#767676;&quot;&gt;Scan the QR code using your google Authenticator App&lt;br/&gt;and enter the OTP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{


border-radius:15px;
}
QPushButton {
        background:transparent;
        border:none;
        border-radius: 20px;
    }
    QPushButton:hover {
        background-color: rgb(206, 206, 206);
    }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/icons/resources/icons8-back-96.png</normaloff>:/icons/resources/icons8-back-96.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>52</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource.qrc">:/icons/resources/logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="error_text">
          <property name="geometry">
           <rect>
            <x>188</x>
            <y>491</y>
            <width>201</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #D9534F;

 </string>
          </property>
          <property name="text">
           <string>Access Key can not be empty</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="error_icon">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>490</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource.qrc">:/icons/resources/icons8-error-50.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
  <include location="../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>widget_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>912</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>844</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>widget_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>935</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>1048</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
